crates/flashblocks/payload/src/generator.rs

```diff
@@ -333,14 +345,28 @@ impl FlashblocksJobGeneratorConfig {
         self.enable_authorization = enable;
         self
     }
+
+    /// Sets the recommitment interval at which the job should re-commit to the transaction pool.
+    pub const fn recommitment_interval(mut self, interval: Duration) -> Self {
+        self.recommitment_interval = interval;
+        self
+    }
+
+    /// Sets the maximum number of concurrent payload build tasks.
+    pub const fn max_payload_tasks(mut self, max: usize) -> Self {
+        self.max_payload_tasks = max;
+        self
+    }
 }
 
 impl Default for FlashblocksJobGeneratorConfig {
     fn default() -> Self {
         Self {
             interval: Duration::from_millis(200),
+            recommitment_interval: Duration::from_millis(200),
             deadline: Duration::from_secs(2),
             enable_authorization: true,
+            max_payload_tasks: 50,
```

Comment:
I'm not sure but isn't 50 way too high? I would think we wouldn't want more tasks than we expect to have threads? Maybe 4 is a sane default?